pkg/util/workloadrepo/snapshot.go

```diff
@@ -218,6 +187,43 @@ func (w *worker) startSnapshot(_ctx context.Context) func() {
 				if err := updateHistSnapshot(ctx, sess, snapID, errs); err != nil {
 					logutil.BgLogger().Info("workload repository snapshot failed: could not update hist_snapshots", zap.NamedError("err", err))
 				}
+
+				continue
+			case <-w.snapshotChan:
+			case <-w.snapshotTicker.C:
+			}
+
+			// coordination logic
+			if !w.owner.IsOwner() {
```

Comment:
If a command is executed in an non-owner node, it will take no effect?

Reply:
Yes. `<- wch` above will take effect on non-owner nodes. However `snapshotTicker` or `snapshotChan` require to be owner.

Reply:
This code will run on both owners and non-owners. Non-owners only execute tasks on `<- wch`.

Reply:
It is expected. But I make non-owner works now.